tap_file/streams.py

```diff
@@ -20,30 +24,51 @@ def get_rows(self) -> Generator[dict[str | Any, str | Any], None, None]:
         Yields:
             A dictionary containing information about a row in a *SV.
         """
-        for reader in self._get_readers():
+        header_skip = self.config["header_skip"]
+        footer_skip = self.config["footer_skip"]
+
+        for reader_dict in self._get_readers():
```

Comment:
This means that the header and footer are being first ran through `DictReader` right? I think this would cause issues for most of the use cases for this.

Reply:
I can see why. I've updated it to process `delimited_header_skip` and `delimited_footer_skip` before parsing with `DictReader`.